scripts/utils/kakarot.py

```diff
@@ -0,0 +1,268 @@
+import json
+import logging
+from pathlib import Path
+from typing import Optional, Union
+
+from eth_account import Account as EvmAccount
+from eth_utils.address import to_checksum_address
+from hexbytes import HexBytes
+from starknet_py.abi import AbiParser
+from starknet_py.contract import Contract
+from starknet_py.net.account.account import Account
+from starknet_py.net.client_errors import ContractNotFoundError
+from starknet_py.net.client_models import Call
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.serialization import serializer_for_function
+from starkware.starknet.public.abi import get_selector_from_name
+from web3 import Web3
+from web3.contract import Contract as Web3Contract
+
+from scripts.constants import (
+    CHAIN_ID,
+    EVM_ADDRESS,
+    EVM_PRIVATE_KEY,
+    GATEWAY_CLIENT,
+    KAKAROT_ADDRESS,
+    KAKAROT_CHAIN_ID,
+)
+from scripts.utils.starknet import deploy_and_fund_evm_address, get_tx_url
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def get_contract(contract_path: str):
+    path = Path(contract_path)
+    solidity_file_path = list(path.parent.glob("**/" + path.stem + ".json"))
+    if len(solidity_file_path) != 1:
+        raise ValueError(f"Cannot locate a unique {path.stem} in {path.name}")
```

Comment:
it seems to me that this is copy/pasted from the current utils without a proper adaptation.

- the naming first remains the old one: `solidity_file_path` is not a path to a solidity file but to a compiled file
- in the test utils, the function takes as input only a contract name and tries to resolves the corresponding contract, here you take already the contract path as input, so there shouldn't be any need for such a path resolution
- indeed we know where the built files are, it is defined in the `foundry.toml` file (this is true for the utils as well that could be updated)

looking at the `get_contract` utils, tbh I don't really understand why it was required to change anything from it https://github.com/sayajin-labs/kakarot/blob/ca1a3ff9886629d88d2cc01c8966ac619440e735/tests/integration/helpers/wrap_kakarot.py#L173

Reply:
- Should have changed the name, oversight from my part.
- The path resolution was used with the idea to allow a user to provide the path to his `.sol` file and find the built files from it.